### DISCUSSION TITLE
How to configure omitLineBreaks with @unhead in Nuxt.js?

### DISCUSSION THREAD

**Discussion by mennghao** [nuxt/nuxt#25303] [Questions] [answered]
I have been exploring the recent update in Nuxt.js version 3.9.1, where some line breaks were removed when rendering HTML, as mentioned in the release notes: nuxt: Remove some line breaks when rendering html.

After testing, I noticed that the removal of line breaks only occurred between <html> and <head>, but there are still line breaks within the <head> tag. Upon further investigation, I found the documentation for @unhead and the omitLineBreaks option: [omitLineBreaks](https://unhead.unjs.io/setup/ssr/installation#omitlinebreaks).

However, I am unsure about how to configure the omitLineBreaks option in Nuxt.js. Could someone provide guidance or an example of how to use this configuration specifically within a Nuxt.js project?

<img width="801" alt="2024-01-19 11 14 53" src="https://github.com/nuxt/nuxt/assets/7802584/353b211d-7add-44c8-b023-c69a9c6abed0">

**Comment by TheAlexLichter** [comment, marked as answer]
I don't think this is configurable right now. https://github.com/nuxt/nuxt/blob/ce7845cbcfea41b63e93feab6c67a06ce654a912/packages/nuxt/src/core/runtime/nitro/renderer.ts#L440C76-L440C89

**Reply by mennghao** [reply]
Thank you!
